controllers/storageoscluster/node-manager_operand.go

```diff
@@ -90,7 +90,7 @@ func (c *NodeManagerOperand) ReadyCheck(ctx context.Context, obj client.Object)
 		return false, err
 	}
 
-	if nodeManagerDep.Status.AvailableReplicas == *nodeManagerDep.Spec.Replicas {
+	if nodeManagerDep.Status.AvailableReplicas > 0 {
 		log.V(4).Info("Found available replicas more than 0", "availableReplicas", nodeManagerDep.Status.AvailableReplicas)
```

Comment:
should this log message now be changed to something like `"Found correct number of available replicas"`?

Reply:
I would address this in a different PR. I just used what we have in other operands.

Reply:
Anyway `Found correct number of available replicas` is not correct, because we found at least one replica. At this point we don't compare desired number with available number. And if we do i should affect the entire system, because of the dependency graph of operands.